test/e2e/README.md

```diff
@@ -9,6 +9,42 @@ To do this, following steps are recommended
 
 ocm get /api/clusters_mgmt/v1/clusters/(cluster-id)/credentials | jq -r .kubeconfig > /(path-to)/kubeconfig
```

Comment:
Not part of this PR, but the sentence above:
```
. Deploy your new version of operator in a test cluster
```

That doesn't seem to apply here.